resources/views/vendor/backpack/theme-tabler/layouts/partials/sidebar_shortcuts.blade.php

```diff
@@ -1,168 +1,8 @@
 <div class="w-100 justify-content-center d-none d-lg-flex sidebar-shortcuts">
     @includeWhen(backpack_theme_config('options.showColorModeSwitcher'), backpack_view('layouts.partials.switch_theme'))
-    <a href="javascript:void(0);" class="nav-link" data-bs-toggle="modal" data-bs-target="#modal-layout">
-        <i class="la la-stream fs-2 text-secondary"></i>
-    </a>
+    <button class="btn btn-link text-secondary nav-link px-0 shadow-none" data-bs-toggle="modal" data-bs-target="#modal-layout">
+        <i class="la la-palette fs-2 me-1"></i>
```

Comment:
I ❤️ that you are using button instead of anchor! GJ 🙏

Reply:
@pxpm You inspired me 😀